cli/roger_build.py

```diff
@@ -82,6 +82,12 @@ def main(self, settingObj, appObj, hooksObj, dockerUtilsObj, dockerObj, args):
             else:
                 repo = data.get('repo', args.app_name)
 
+            build_args = {}
+            if 'build-args' in data:
+                if 'environment' in data['build-args']:
```

Comment:
So trust the user. Got it. There is not way to tell what image was built for which environment tho, right?